Event trigger for ingest-gisaid

ttung opened the pull request:
### Description
event and target for triggering the ingest-gisaid workflow.

Depends on #228 

#### Issue
[ch126779](https://app.clubhouse.io/genepi/story/126779/cloudwatch-trigger-to-invoke-gisaid-workflow)

### Test plan
Changed it to every 15 minutes, and watched it kick off a build.

phoenixAja commented on terraform/events.tf:
```diff
@@ -0,0 +1,43 @@
+    job_definition = aws_batch_job_definition.aspen-batch-job-definition.name
+  }
+
+  input = <<-CONTAINER_OVERRIDES
```
can you add a comment for what this is doing? I'm not familiar with `CONTAINER_OVERRIDES`

ttung replied:
done.

phoenixAja reviewed (approved):
exciting looks good!

Commit:
Event trigger for ingest-gisaid

### Description
event and target for triggering the ingest-gisaid workflow.

#### Issue
[ch126779](https://app.clubhouse.io/genepi/story/126779/cloudwatch-trigger-to-invoke-gisaid-workflow)

### Test plan
Changed it to every 15 minutes, and watched it kick off a build.
